.github/workflows/pr.yaml

```diff
@@ -44,10 +44,14 @@ jobs:
           do
             sleep 1
           done
-      - name: Deploy konk-service
+      - name: Setup Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.x
+      - name: Deploy Test APIServer
         timeout-minutes: 5
         run: |
-          make deploy-konk-service
```

Comment:
Since this is deployed later by the operator, it's a waste of time to deploy it here too.

Reply:
True, I included it because we deploy both the konk chart and konk CR (along with the operator), but since we are not running any test on the konk-service chart, it may not be necessary.